tests/wpt/web-platform-tests/html/rendering/non-replaced-elements/tables/table-width.html

```diff
@@ -0,0 +1,25 @@
+<link rel="match" href="table-width-ref.html">
```

Comment:
Yes, failing for unrelated reasons; as a workaround, you could try "border-spacing: 0;".